packages/notionable/src/global/logic.ts

```diff
@@ -56,11 +57,35 @@ export async function onAccountPending(
     );
   });
 
+  let hasContestNFT;
+  let contestTokenId;
+
+  const NFT = '0x7c2d3a5fa3b41f4e6e2086bb19372016a7533f3e';
```

Comment:
Can you refactor the `useNFTContract` to remove the logic since it has moved here?